client/state/themes/constants.js

```diff
@@ -6,3 +6,56 @@ export const DEFAULT_THEME_QUERY = {
 	offset: 0,
 	page: 1
 };
+
```

Comment:
Maybe add a comment here what Premium Squared means, and that this list should be in sync with... some other file on WPCOM? 😄